backend/src/Api/App/Controller/InitController.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace App\Api\App\Controller;
+
+use App\Entity\Publication;
+use App\Repository\ItemRepository;
+use App\Repository\PublicationRepository;
+use App\Repository\CollectionRepository;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Annotation\Route;
+
+#[Route('/init', name: 'app_api_init_')]
+class InitController extends AbstractController
+{
+    public function __construct(
+        private readonly ItemRepository $itemRepository,
+        private readonly PublicationRepository $publicationRepository,
+        private readonly CollectionRepository $collectionRepository,
+    ) {
+    }
+
+    #[Route('', name: 'default', methods: ['GET'])]
+    public function getInit(Request $request): JsonResponse
+    {
+        $data = [
+            "collections" => [],
+            "publications" => [],
+            "items" =>  [],
+            "selectedCollection" => null,
+            "selectedPublication" => null
+        ];
+
+        $collections = $this->collectionRepository->findAll();
+        foreach ($collections as $collection) {
+            $data["collections"][] = [
```

Comment:
Use DTO objects instead.

See [how we map entities to DTO objects](https://github.com/hyvor/post/blob/main/backend/src/Api/Console/Controller/DomainController.php#L37)
Here's an example [DTO object](https://github.com/hyvor/post/blob/main/backend/src/Api/Console/Object/DomainObject.php)